README.md

```diff
@@ -3,18 +3,48 @@
 [http://colinxfleming.github.io/dcaf_case_management/](http://colinxfleming.github.io/dcaf_case_management/)
 
 ## Project description
-This project is a case management system. 
+This project is a case management system for the [DC Abortion Fund](http://dcabortionfund.org/), an all-volunteer, 501(c)(3) non-profit organization that gives grants to people in DC, Maryland, and Virginia who cannot afford the full cost of abortion care. Currently, a team of around 75 case managers are taking about 3,500 calls a year and entering them all into shared Excel sheets. We're replacing that with a nice, clean, usable and scalable rails application! This will let DCAF continue to operate at a fast pace, and prevent volunteers from getting frustrated with shared Excel sheets. 
 
-We're partnering with the [DC Abortion Fund](http://dcabortionfund.org/), an all-volunteer, 501(c)(3) non-profit organization that makes grants to women and girls in DC, Maryland, and Virginia who cannot afford the full cost of an abortion.
+If you're interested: This is a great opportunity to learn about common nonprofit data problems, try out Rails, or just contribute to a project that will have a great positive impact on reproductive justice for people in the DC area. Come say hi, we're friendly! 
 
-## Contributing
-* To contribute, visit the main project page and fork from the Master Branch.
-* In your terminal, create a directory and use `git clone` to store the files locally on your computer.
-* When you have made changes and you want to upload them onto Github, add and commit your changes by using the following commands: `git add` and `git commit`
-* Push to origin master by inputting `git push -u origin master`
-* Once you have pushed your changes, make a pull request.
+We're generally looking for people comfortable or interested in the following:
+* Ruby on Rails (a good starter kit: [CodeAcademy's course](http://www.codecademy.com/learn/learn-rails))
+* JQuery / HTML / CSS
+* NoSQL / flat data modeling
+* Nonprofit data issues 
+
+Get started with the how-and-why of the project by [checking out DCAF](http://dcabortionfund.org), checking out [DCAF Case Manager Lisa's explanation of DCAF's business logic](TK), looking at the design team's [InVision prototype](https://projects.invisionapp.com/share/6757W6WFJ), and reading the `#dcaf_case_management` channel on Slack. Hit up @colinxfleming and Mollie with any questions. 
```

Comment:
Yeah, it's the Mollie from Slack. Wanted to ask her before I linked her in the readme.

The TK is a doc we're ironing out still to provide a 101-level overview of DCAF. I'm throwing the draft version in the wiki shortly, thank you for this reminder to update the link!